index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>gbenga-portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="nav-container">
+            <nav class="nav-bar">
+                <header>
+                    <h1><a href="#">Gbenga Oyewunmi</a></h1>
+                </header>
+                <img src="./images/Union.png" alt="">
+            </nav>
+        </div>
```

Comment:
- [ ] Kindly make the `header` tag the outer container for semantic purposes so it will be like:
```
<header>
      <nav>
           your code ....................
      </nav>
</header>
```